### PR TITLE
chore: add v13 welcome page content

### PR DESCRIPTION
**Pull request by AtofStryker**
- Closes N/A

### Additional details

Adds the v13 welcome/splash page with updated content

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by jennifer-shehane — APPROVED** [review]
Content looks good, I didn't do a thorough code review and would like someone to do that and get it up and running in their review.

**Comment by AtofStryker** [comment]
> Content looks good, I didn't do a thorough code review and would like someone to do that and get it up and running in their review.

Sounds good. The `v13` video is in staging currently, so whoever tests this would need to run:
```
CYPRESS_INTERNAL_ENV=staging yarn cypress:open
```

inside the `packages/launchpad`

**Comment by marktnoonan** [comment]
I'll plan to pull this down later today and review!

**Comment by AtofStryker** [comment]
@tgriesser I want to double check and make sure I handled the resolution properly in the graphQL resolver.

File: packages/launchpad/src/Main.vue
Comment (ryanthemanuel) [inline packages/launchpad/src/Main.vue:13]: Will this pass if videoHtml is ''?

File: system-tests/projects/migration-v12-to-v13/cypress.config.js
Comment (ryanthemanuel) [inline system-tests/projects/migration-v12-to-v13/cypress.config.js:1]: Where do these get run?

File: packages/launchpad/src/migration/MajorVersionWelcome.vue
Comment (marktnoonan) [inline packages/launchpad/src/migration/MajorVersionWelcome.vue]:
```suggestion
                  changelog</ExternalLink>. To take a deep-dive into this release, read our <ExternalLink href="https://on.cypress.io/cypress-13-release?utm_source=splash-page&utm_campaign=v13">
                  <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
                  blog post</ExternalLink>.
```

In order to prevent the space before the period, we need to not have a newline before the closing tag.

File: packages/launchpad/src/migration/MajorVersionWelcome.vue
Comment (marktnoonan) [inline packages/launchpad/src/migration/MajorVersionWelcome.vue]: Generally we use only 2 values for `utm_source` - `Binary: App` or `Binary: Launchpad`. Is this intentionally different from that? I realize it's not documented well. There's a [sheet](https://docs.google.com/spreadsheets/d/1Cs933TLXcS63PmMlXA4wKLJtQSFH90dProvzI9oyEB4/edit#gid=0) that wrangles these values, it would be good to add these there. Medium seems more suitable than "source" for "splash page" based on the existing pattern.

But also curious - is there a particular person expecting to do some analysis later based on these params, or are we more playing it safe?

File: packages/launchpad/src/Main.vue
Comment (AtofStryker) [reply packages/launchpad/src/Main.vue:13]: It does not. `v-if` only evaluates with the expression is truthy https://vuejs.org/guide/essentials/conditional.html#v-if

File: packages/launchpad/src/migration/MajorVersionWelcome.vue
Comment (marktnoonan) [inline packages/launchpad/src/migration/MajorVersionWelcome.vue]:
```suggestion
                  <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
                  documentation</ExternalLink>.
```

Or we could just disable that whole. It's great for regular html, silly for rich text.

File: system-tests/projects/migration-v12-to-v13/cypress.config.js
Comment (AtofStryker) [reply system-tests/projects/migration-v12-to-v13/cypress.config.js:1]: in the e2e migration test https://github.com/cypress-io/cypress/blob/fca2b379f3c2bae1eefb37b5a7f26fcd6e2426c4/packages/launchpad/cypress/e2e/migration.cy.ts#L1736

File: packages/launchpad/src/migration/MajorVersionWelcome.vue
Comment (marktnoonan) [inline packages/launchpad/src/migration/MajorVersionWelcome.vue]: What would you think of using `<InlineCodeFragment>` here instead of plain `code` elements. It has a little bit better visual separation from the surrounding text and looks more like what we do elsewhere.

Comment (AtofStryker) [reply packages/launchpad/src/migration/MajorVersionWelcome.vue]: I can update the references and push it up!

File: packages/launchpad/src/migration/MajorVersionWelcome.vue
Comment (AtofStryker) [reply packages/launchpad/src/migration/MajorVersionWelcome.vue]: I'm probably not the best person to ask about the `utm_source`, but it was in the approved copy for the splash page. I wonder if it makes sense to update the `utm_source` to `Binary: App` and leverage `medium` for the `splash page` . Heres the [copy](https://docs.google.com/document/d/1LjVmCnM1KTM9Uu6TqZzagCD60KRQs1gaWb3kzhJQv08/edit) for reference. I can ask the GTM team in slack

Comment (marktnoonan) [reply packages/launchpad/src/migration/MajorVersionWelcome.vue]: Good idea to clarify in Slack. Being consistent is nice but there's not a hard and fast rule.

File: packages/launchpad/src/migration/MajorVersionWelcome.vue
Comment (AtofStryker) [reply packages/launchpad/src/migration/MajorVersionWelcome.vue]: IMO I find it cleaner

**Review by marktnoonan — APPROVED** [review]
This all looks good to me, approving with the assumption that we figure out the UTM params before merge.

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**
